src/wallet/wallet.cpp

```diff
@@ -4010,6 +4010,11 @@ bool CWallet::ParameterInteraction()
         LogPrintf("%s: parameter interaction: -salvagewallet=1 -> setting -rescan=1\n", __func__);
     }
 
+    // -zapwallettx implies dropping the mempool on startup
+    if (GetBoolArg("-zapwallettxes", false) && SoftSetBoolArg("-persistmempool", false)) {
```

Comment:
"=\<mode\>" seems like a super strange string when \<mode\> isnt replaced out in the logs (plus its a boolean, what does \<mode\> mean?). zapwallettxes looks like the only one where it looks like this.